src/main/java/play/dev/filewatch/FileWatchService.java

```diff
@@ -18,6 +20,12 @@ public interface FileWatchService {
    */
   FileWatcher watch(Iterable<File> filesToWatch, Runnable onChange);
 
+  /** @deprecated Use {@link #watch(Iterable, Runnable)} instead */
+  @Deprecated
+  default FileWatcher watch(List<File> filesToWatch, Callable<Void> onChange) {
+    return watch(filesToWatch, () -> onChange.call());
+  }
```

Comment:
Stay compatible with current versions, because why not? I want to avoid breaking Play users as much as possible.